byway/src/main/webapp/routepage.js

```diff
@@ -49,27 +63,139 @@ function calcRoute(directionsService, directionsRenderer, start, end) {
   });
 }
 
+/** Get the new list of stops from datastore onload */
+function getStops(){
+  const stopList = document.getElementById('stop-list');
+
+  if(stopList != null){
+    stopList.innerText = ""; // clear list
+  }
+
+  fetch('/api/stop')
+  .then(response => response.json())
+  .then((stopsResponse) => {
+    stopsResponse.forEach((stop)=>{
+      stops.push(stop);
+      var btn = document.createElement('button');
+      btn.id = stop.id;
+      btn.innerText = stop.placename;
+      btn.setAttribute("class", "btn rec-btn");
```

Comment:
To remain consistent, consider accessing/setting the attribute by `btn.className = "btn rec-btn"` rather than using `setAttribute`

Reply:
Oh no... in my master stuff I also had this format. I'll make a note to change that, but this is changed.